### PR TITLE
Kustomize

### PR DESCRIPTION
**Pull request by twerthi**
Add kustomize option to octopub project.

### CODE REVIEW

File: kustomize/envs/Development/secrets.properties [hunk 1]
```diff
@@ -0,0 +1,2 @@
+database-password=#{database-password}
```
Comment (harrisonmeister) [inline kustomize/envs/Development/secrets.properties]: Is there any reason these variables couldnt follow the `.` notation used for other variable substitution? 

e.g. `#{Project.Database.Password}`

File: kustomize/envs/Development/variables.properties [hunk 2]
```diff
@@ -0,0 +1,5 @@
+DATABASE_HOSTNAME=#{DATABASE_HOSTNAME}
```
Comment (harrisonmeister) [inline kustomize/envs/Development/variables.properties]: Is there any reason these variables couldnt follow the `.` notation used for other variable substitution? 

e.g. `#{Project.Database.Hostname}`

**Review by harrisonmeister — APPROVED** [review]
LGTM. I've made a couple suggestions on the octostache variable names

File: kustomize/envs/Development/variables.properties (hunk 2 above)
Comment (twerthi) [reply kustomize/envs/Development/variables.properties]: This variables were originally going to be used for Structured Configuration Variables ... until I found that wasn't enabled on Kustomize step.

Comment (twerthi) [reply kustomize/envs/Development/variables.properties]: Variables updated.

**Review by harrisonmeister — APPROVED** [review]
LGTM

**Pull request merged**